community/kernel/src/main/java/org/neo4j/kernel/internal/GraphDatabaseAPI.java

```diff
@@ -48,6 +52,43 @@
      */
     URL validateURLAccess( URL url ) throws URLAccessValidationError;
 
-    @Deprecated
     String getStoreDir();
+
+    /**
+     * Returns all relationship types currently in the underlying store.
+     * Relationship types are added to the underlying store the first time they
+     * are used in a successfully commited {@link Node#createRelationshipTo
+     * node.createRelationshipTo(...)}. Note that this method is guaranteed to
+     * return all known relationship types, but it does not guarantee that it
+     * won't return <i>more</i> than that (e.g. it can return "historic"
+     * relationship types that no longer have any relationships in the node
+     * space).
+     *
+     * @return all relationship types in the underlying store
+     */
+    ResourceIterable<RelationshipType> getAllRelationshipTypesInUseAndNot();
```

Comment:
you have 2 places where methods does not represent what they actually do - so their names are confusing and inconsistent with actual behaviour, so that is more then a valid reason to rename them. Also as we already figured out, nothing preventing us from doing it. Thats why i think is a good idea.